src/components/LightSwitch.js

```diff
@@ -1,11 +1,45 @@
 import React from "react"
+import { useState } from "react"
+import switchOn from '../assets/switchOn.png'
+import switchOff from '../assets/switchOff.png'
+import lightOn from '../assets/lightOn.png'
+import lightOff from '../assets/lightOff.png'
 
 const LightSwitch = () => {
+  const [lightSwitchStatus, setLightSwitchStatus] = useState(false)
+
+  const turnLight = () => {
+    if (lightSwitchStatus === false){
+      setLightSwitchStatus(true)
+    } else {
+      setLightSwitchStatus(false)
+    }
+  }
+
+  let lightStatus = ""
+  if (lightSwitchStatus === false) {
+    lightStatus = "Off"
+  } else {
+    lightStatus = "On"
+  }
+
   return (
     <>
-      <div className="lightbulb"></div>
+      <div className="bulbWrap">
+        <div 
+          onClick={turnLight} 
+          className="lightbulb" 
+          style={{backgroundImage: lightSwitchStatus ? 'url(' + switchOn + ')' : 'url(' + switchOff + ')', backgroundSize: 'contain'}}
+        >
+          {lightStatus}
+        </div>
+
+        {!lightSwitchStatus && <div className="lightOffPic" style={{backgroundImage: 'url(' + lightOff + ')', backgroundSize: 'contain'}}></div>}
+
+        {lightSwitchStatus && <div className="lightOnPic" style={{backgroundImage: 'url(' + lightOn + ')', backgroundSize: 'contain'}}></div>}
+      </div>
     </>
```

Comment:
This looks good.  I like how you kept the logical component separate from your display component.  Again, this makes it easy to read and understand your intentions with your code.